docs/src/modules/components/AppFrame.js

```diff
@@ -90,7 +90,7 @@ const styles = theme => ({
     padding: 4,
     textAlign: 'center',
     backgroundColor: theme.palette.primary.dark,
-    color: theme.palette.getContrastText(theme.palette.primary.dark),
+    color: theme.palette.primary.contrastText,
```

Comment:
For people wondering
```jsx
color: theme.palette.primary.contrastText,
```
is equivalent to 
```jsx
color: theme.palette.getContrastText(theme.palette.primary.main),
```

So it skewed the contrast output contrast color in one direction.